en/docs/choreo-cli/using-personal-access-tokens.md

```diff
@@ -0,0 +1,125 @@
+# Personal Access Tokens Guide for Choreo CLI
```

Comment:
Can we go with "Manage Authentication with Personal Access Tokens" as the title?
WDYT?